eva_submission/submission_qc_checks.py

```diff
@@ -268,6 +284,154 @@ def check_if_browsable_files_entered_correctly_in_db(self, vcf_files):
             expected files: {vcf_files}
             missing files: {missing_files if missing_files else 'None'}"""
 
+    def clustering_check_report(self, target_assembly):
+        clustering_log_file = glob.glob(f"{self.path_to_data_dir}/53_clustering/logs/{target_assembly}_clustering.log")
+        clustering_qc_log_file = glob.glob(
+            f"{self.path_to_data_dir}/53_clustering/logs/{target_assembly}_clustering_qc.log")
+
+        clustering_error = ""
+        if clustering_log_file:
+            if not self.check_if_job_completed_successfully(clustering_log_file[0], 'clustering'):
+                clustering_error += f"failed_job - {self.get_failed_job_name(clustering_log_file[0])}"
+                clustering_check_result = "FAIL"
+            else:
+                clustering_check_result = "PASS"
+        else:
+            clustering_error += f"Clustering Error : No clustering file found for {target_assembly}_clustering.log"
+            clustering_check_result = "FAIL"
+
+        clustering_qc_error = ""
+        if clustering_qc_log_file:
+            if not self.check_if_job_completed_successfully(clustering_qc_log_file[0], 'clustering_qc'):
+                clustering_qc_error += f"failed_job - {self.get_failed_job_name(clustering_qc_log_file[0])}"
+                clustering_qc_check_result = "FAIL"
+            else:
+                clustering_qc_check_result = "PASS"
+        else:
+            clustering_qc_error += f"Clustering QC Error : No clustering qc file found for {target_assembly}_clustering_qc.log"
+            clustering_qc_check_result = "FAIL"
+
+        if clustering_check_result == 'FAIL' or clustering_qc_check_result == 'FAIL':
+            self._clustering_check_result = 'FAIL'
+        else:
+            self._clustering_check_result = 'PASS'
+
+        return f"""
+                    Clustering Job: {clustering_check_result}        
+                        {clustering_error if clustering_check_result == 'FAIL' else ""}
+                    Clustering QC Job: {clustering_qc_check_result}
+                        {clustering_qc_error if clustering_qc_check_result == 'FAIL' else ""}
+        """
+
+    def remapping_check_report(self, target_assembly):
+        failed_asm = {}
+        for analysis_data in self.analyses.values():
+            assembly_accession = analysis_data['assembly_accession']
+            if assembly_accession != target_assembly:
```

Comment:
`vcf_extractor_result` won't be initiated if the `assembly_accession == target_assembly` 
I would prefer starting with something like 
```suggestion
            vcf_extractor_result = remapping_ingestion_result = 'SKIP'
            if assembly_accession != target_assembly:
```

Reply:
If we do this, it will put the status of each assembly in the report regardless of pass, fail or skip. Do we want that ?
Currently it shows status for each failed assemblies. For others, it just shows PASS

Reply:
We won't fail the QC if remapping is skipped but we might want to highlight it in the report. I think you could put the status of each analysis in the report.

Reply:
done